index.html

```diff
@@ -15,9 +15,40 @@
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-  <header>
+  <header class="header-background">
+    <div class="header">
+        <div class="row column-full">
+      <h3 class="row header libre-font">Code Journal</h3>
+      </div>
+    </div>
   </header>
-  <main>
+  <main class="entry-background libre-font one-rem-padding">
+    <div data-view="entry-form">
+      <form id="entry">
+        <div class="row column-full zero-one-and-half-padding">
+          <h2 class="title">New Entry</h2>
+        </div>
+        <div class="row one-rem-padding">
+          <div class="column-half">
+            <label for="image"></label>
+            <img src="images/placeholder-image-square.jpg">
+          </div>
+          <div class="column-half libre-font">
+            <label for="entry-title">Title</label>
+            <input required name="title" type="text" id="entry-title">
+            <label for="entry-photo-url">Photo URL</label>
+            <input required name="photoUrl" type="url" id="entry-photo-url">
+          </div>
+          <div class="column-full">
+            <label for="entry-notes">Notes</label>
+            <textarea required rows = "6" name="notes" id="entry-notes"></textarea>
+          </div>
+        </div>
+        <div class=" row column-full flex-end zero-one-and-half-padding">
```

Comment:
No element should be both a row and a column, make this into a `row` element with a `column` element as its child. I see a couple other elements that are both `row` and `column` elements too, please revise those as well